autogpt_platform/backend/backend/blocks/github/issues.py

```diff
@@ -62,6 +68,10 @@ def __init__(self):
     def post_comment(
         credentials: GithubCredentials, issue_url: str, body_text: str
     ) -> tuple[int, str]:
+
+        if is_github_url(issue_url) is False:
+            raise ValueError("The input URL must be a valid GitHub URL.")
+
```

Comment:
What if we update input to be numeber/ string of the relevant inputs instead? @majdyz

Reply:
Any ideas in mind? what the user need to provide?